Lib/subprocess.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *, dwFlags=0, hStdInput=None, hStdOutput=None,
             self.hStdOutput = hStdOutput
```

Comment:
I tried to add some more documentation about this but I'm not really sure it's the right place for it. Feel free to suggest better wording or a better place for it.